### ISSUE TITLE
Path handling regression with SASS

### ISSUE THREAD

**Issue by esuomi** [mvnpm/esbuild#6]
Hi,

Unfortunately the details I can provide are somewhat hazy but from what I can tell, the problem I have occurs specifically in this wrapper library and is potentially a regression between [v0.0.8 and v0.0.9](https://github.com/mvnpm/esbuild/compare/v0.0.8..v0.0.9).

I have a pretty straightforward problem: after updating Quarkus Web Bundler from 1.7.2 to 1.7.3 all SCSS processing started to fail like so:
```shell
? [ERROR] exec: "/var/folders/nj/fnq0l71j37v94vqm0241gn9r0000gq/T/esbuild-0.23.0-mvnpm-0.0.9/dart-sass/sass": stat /var/folders/nj/fnq0l71j37v94vqm0241gn9r0000gq/T/esbuild-0.23.0-mvnpm-0.0.9/dart-sass/sass: no such file or directory [plugin sass-loader]

    main.js:2:7:
      2 ? import "./app/styles.scss";
        ?        ~~~~~~~~~~~~~~~~~~~

1 error
[watch] build finished, watching for changes...
        at io.mvnpm.esbuild.Execute$Streamer.lambda$run$0(Execute.java:173)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
```

the reason I'm reporting this issue here is because of transitive dependencies:

 - Quarkus Web Bundler's only change in [1.7.2->1.7.3 point release](https://github.com/quarkiverse/quarkus-web-bundler/compare/1.7.2..1.7.3) which relates to this functionality was updating `mvnpm/esbuild-java-bundle-scss` from v1.5.0 to v1.5.1, see also https://github.com/quarkiverse/quarkus-web-bundler/pull/282
 - [mvnpm/esbuild-java v1.5.0 -> v1.5.1](https://github.com/mvnpm/esbuild-java/compare/v1.5.0..v1.5.1) merely updates its `mvnpm/esbuild` dependency version from v0.0.8 to v0.0.9
 - as far as I can tell, your patch release changes the logic how paths are handled, and while this seems to work for some, it now broke the build for me

For context, I use a Mac with some additional enterprisey dictated protections, so I'm suspecting this might simply be a path permission issue, but pinpointing is kinda hard.

...admittedly somewhat of a mess.

**Comment by edewit** [comment]
Right, what we did was to improve import handeling as we now also find paths without extensions in the node folder. On top of that we now watch all the included files. To better debug the issue you could run esbuild from the command line
